helpers/redmine_helper.rb

```diff
@@ -37,4 +37,12 @@ def is_http_get_response_positive?(url)
     end
   end
 
+  def log_out
+    begin
+      @browser.find_element(css: '.logout').click
+    rescue Selenium::WebDriver::Error::NoSuchElementError
```

Comment:
Hi @d-konovalov, thank you for you review)

This method could be called when user is not logged in in some cases so I've just added return here to have some meaningful result action (previously the message `User is already logged out` was returned, but it was spoiling reports all the time)